main/src/examples/ABMAugmentionExample/ABMAugmentionExample.cpp

```diff
@@ -187,22 +191,16 @@ void ABMAugmentionExample::augmentImages() {
         //imshow("Canny", augmented);
         //waitKey(50);
 
-        // convert back to IplImage
-        IplImage augmentedImageIpl = augmented;
-        // end your stuff
-        // from IplImage to yarp image
-        cvCvtColor(&augmentedImageIpl, &augmentedImageIpl, CV_BGR2RGB);
         ImageOf<PixelRgb> augmentedImageYarp;
-        augmentedImageYarp.resize(augmentedImageIpl.width, augmentedImageIpl.height);
-        cvCopyImage(&augmentedImageIpl, (IplImage *)augmentedImageYarp.getIplImage());
+        augmentedImageYarp.resize(vRawImages[i]);
+        Mat img3_mat=cvarrToMat((IplImage*)augmentedImageYarp.getIplImage());
+        cvtColor( augmented, img3_mat, CV_BGR2RGB );
```

Comment:
The `cvarrToMat()` function just provides a wrapper on the yarp image; thus, in this case, it doesn't create a new image.

Therefore, I think the operations are correct.